### ISSUE TITLE
Grid breaking with monospace fonts?

### ISSUE THREAD

**Issue by ryan-parag** [daneden/Toast#56]
Not sure if anyone experienced this, but the grid seems to be breaking on me when font is styled to `monospace`.

[Example in CodePen](https://codepen.io/ryanparag/pen/35650aa2a9c364b01844d632c278ce99)

Am I missing something? Or has anyone had experience in fixing this issue?

**Comment by Enweave** [comment]
**Tl;dr:**
In your pen you could change scss on line 17 like this:
```
body {
	font-family: system-ui, sans-serif;
	&.u-font--monospace .demo-item{
		font-family: monospace, monospace;
		font-size: 1em;
	}
}
```

**But why?**:
At the bottom it's a "[spaces between inline-blocks](https://davidwalsh.name/remove-whitespace-inline-block)" problem, which does not have solid solution, therefore many new grids use flex or whatever. 
However Toast grid, being inline-block grid:), uses [negative margin](https://github.com/daneden/Toast/blob/master/scss/_grid.scss#L186) to partially mitigate the issue. And, as I think, when you switch body font to "monospace" the width of a space between blocks is being changed _differently_, so the existing negative margin value no longer works...
